notebooks/demo3/demo3-create-tables.ipynb

```diff
@@ -0,0 +1,546 @@
+{
```

Comment:
<p>Can this be done using Erik's <code>osc-ingest-tools</code>  library, or are there custom changes here that are not included in it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>We havent tried <code>osc-ingest-tools</code> yet, will  experiment with that and try to leverage that here. We might also add that as an enhancement for a forthcoming PR if it seems to be taking some additional efforts to customize it</p>

Reply:
Sounds good! Yeah we can definitely add it as an enhancement in a future PR as well :)